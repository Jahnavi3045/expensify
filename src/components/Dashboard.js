import React from 'react'
// import { useDispatch, useSelector } from 'react-redux'
// import { useDispatch} from 'react-redux'
// import {addExpense} from '../actions/expenses'
// import {setTextFilter} from '../actions/filters'
// import getVisibleExpenses from '../selectors/expenses'
import ExpenseList from './ExpenseList'
import FilterExpenseList from './FilterExpenseList'
import ExpensesSummary from './ExpensesSummary'

const Dashboard = () => {
  // const dispatch=useDispatch();
  // dispatch(addExpense({description:'water bill',amount:355,createdAt:110}));
  // dispatch(addExpense({description:'gas bill',createdAt:100,amount:1000}));
  // dispatch(addExpense({description:'wifi bill',createdAt:35,amount:200}));
  // dispatch(addExpense({description:'house rent',createdAt:0,amount:6000}));
  // dispatch(setTextFilter('gas'));
  
  // const state=useSelector((state)=>state)
  // const visibleExpenses=getVisibleExpenses(state.expenses,state.filters)
  // console.log(visibleExpenses)

  return (
    <div>
      <FilterExpenseList/>
      <ExpenseList/>
      <ExpensesSummary/>
    </div>
  )
}

export default Dashboard
