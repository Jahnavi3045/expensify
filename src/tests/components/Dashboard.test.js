import React from "react";
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Dashboard from "../../components/Dashboard";

test('should render dashboard correctly',()=>{
    const renderer=new ReactShallowRenderer()
    renderer.render(<Dashboard/>)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})