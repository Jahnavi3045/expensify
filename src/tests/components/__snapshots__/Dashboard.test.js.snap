// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard correctly 1`] = `
<div>
  <FilterExpenseList />
  <Memo(Connect(ExpenseList)) />
  <Memo(Connect(ExpensesSummary)) />
</div>
`;
